Code/Source/RGLSystemComponent.cpp

```diff
@@ -109,12 +113,26 @@ namespace RGL
 
     void RGLSystemComponent::OnEntityContextCreateEntity(AZ::Entity& entity)
     {
-        if (entity.FindComponent(AZ::Render::MeshComponentTypeId) == nullptr || m_excludedEntities.contains(entity.GetId()))
+        if (m_excludedEntities.contains(entity.GetId()))
         {
             return;
         }
 
-        [[maybe_unused]] bool inserted = m_entityManagers.emplace(entity.GetId(), EntityManager{ entity.GetId() }).second;
+        AZStd::shared_ptr<EntityManager> entityManager;
+        if (entity.FindComponent<EMotionFX::Integration::ActorComponent>())
+        {
+            entityManager = AZStd::make_shared<ActorEntityManager>(entity.GetId());
+        }
+        else if (entity.FindComponent(AZ::Render::MeshComponentTypeId))
+        {
+            entityManager = AZStd::make_shared<MeshEntityManager>(entity.GetId());
+        }
+        else
+        {
+            return;
```

Comment:
Error log?

Reply:
This is a valid case, where the newly created entity does not interest us.